client/src/components/App/App.tsx

```diff
@@ -49,9 +49,9 @@ const App: FC<unknown> = () => {
     }
   }, [testSuites]);
 
-  function handleResize() {
+  const handleResize = () => {
     setWindowIsSmall(window.innerWidth < 1000);
```

Comment:
About moving to the fat arrow function ... are you running into binding issues?  I've run into a couple.

Reply:
Haven't seen any binding issues yet, might be worth investigating if it keeps happening?

Reply:
So this is a style/consistency change?  (I mean, I prefer fat arrow functions by default ... just curious)

Reply:
For consistency!